kfp-tekton/prometheus/kustomization.yaml

```diff
@@ -0,0 +1,2 @@
+resources:
```

Comment:
imo the kustomization file in `kfp-tekton/base` should have a reference to this folder too.

Reply:
updated the kustomization file in `kfp-tekton/base` to have a reference to this folder.